### PR TITLE
BL-1298: Do not count article bookmarks.

### PR DESCRIPTION
**Pull request by dkinzer**
* Also removes more unnecessary article bookmark code.
* Fixes one styling issue that was causing font to be white in select button.

### CODE REVIEW

**Review by sensei100 — CHANGES_REQUESTED** [review]
@dkinzer This looks good, but there is one css issue on mobile devices with the send to menu. This is happening on production too.
Screen Shot 2020-12-10 at 9 17 07 AM

**Comment by dkinzer** [comment]
@sensei100 since that issue is happening on production too, should we create a new ticket to resolve that one?

**Comment by sensei100** [comment]
@dkinzer My opinion is that since this is an issue directly related to the work in this ticket, it makes sense to address it here.  Up to you.

**Comment by dkinzer** [comment]
@sensei100 what device and browser has this issue plese.

**Comment by sensei100** [comment]
@dkinzer I was using the chrome dev tools and clicking on the toggle device button

**Comment by dkinzer** [comment]
@sensei100 I did that and I was not able to reproduce issue.  I was going to try browserstack next.

**Comment by sensei100** [comment]
@dkinzer I see it on my iPhone 10 and I also see it when using safari's responsive mode.

File: Gemfile.lock [hunk 1]
```diff
@@ -451,7 +451,7 @@ GEM
       rspec-mocks (~> 3.9)
       rspec-support (~> 3.9)
     rspec-support (3.10.0)
-    rubocop (1.5.2)
```
Comment (dkinzer) [inline Gemfile.lock:454]: confused by this since we already have that upstream https://github.com/tulibraries/tul_cob/pull/2321/files

**Review by sensei100 — APPROVED** [review]

**Pull request merged**